website/screens/principles/themes/ThemesPage.tsx

```diff
@@ -1427,7 +1427,7 @@ const sections = [
                     <DxcParagraph>
                       More examples can be found in the{" "}
                       <Link
-                        href="/utilities/halstackProvider/#opinionated-theme"
+                        href="/utilities/halstack-provider/#opinionated-theme"
                         passHref
                       >
```

Comment:
When using this link, Halstack provider is not shown as selected in Sidenav.

![image](https://user-images.githubusercontent.com/10975289/225686617-156c5248-cdd1-4914-a520-175245225319.png)

Reply:
It looks like this happens with every relative link that includes a quicknav link (with "#"). We need to fix this.